cmd/zt_interceptors_for_test.go

```diff
@@ -83,7 +85,7 @@ type mockedLifecycleManager struct{}
 
 func (mockedLifecycleManager) Progress(common.OutputBuilder)                            {}
 func (mockedLifecycleManager) Init(common.OutputBuilder)                                {}
-func (mockedLifecycleManager) Info(string)                                              {}
+func (mockedLifecycleManager) Info(msg string)                                          { fmt.Println(msg) }
```

Comment:
Is this supposed to be here permanently? (Only asking because the other method bodies are all empty)

Reply:
yah this is just for testing, as we often output messages when some folder is skipped